### PR TITLE
PMM-6765 Handle schema names.

### PR DESCRIPTION
**Pull request by BupycHuk**
https://github.com/Percona-Lab/pmm-submodules/pull/1169

### CODE REVIEW

File: actions/postgresql_show_create_table_action.go [hunk 1]
```diff
@@ -131,14 +131,24 @@ func (a *postgresqlShowCreateTableAction) Run(ctx context.Context) ([]byte, erro
...
+	row := db.QueryRowContext(ctx, fmt.Sprintf(`SELECT /* pmm-agent */  c.oid,
 	       n.nspname,
 	       c.relname
 	FROM pg_catalog.pg_class c
 	         LEFT JOIN pg_catalog.pg_namespace n ON n.oid = c.relnamespace
 	WHERE c.relname = $1
-	  AND pg_catalog.pg_table_is_visible(c.oid)
-	ORDER BY nspname, relname;`, a.params.Table)
+	  AND pg_catalog.pg_table_is_visible(c.oid) %s
+	ORDER BY nspname, relname;`, namespaceQuery), args...)
```
Comment (askomorokhov) [inline actions/postgresql_show_create_table_action.go:151]: can we move this multiline query string to constant? This should simplify code reading.

**Review by askomorokhov — APPROVED** [review]
Looks good to me.

**Review by JiriCtvrtka — APPROVED** [review]

File: actions/postgresql_show_index_action_test.go [hunk 2]
```diff
@@ -58,4 +58,28 @@ func TestPostgreSQLShowIndex(t *testing.T) {
...
+			Dsn:   dsn,
+			Table: "public.city",
+		}
+		a := NewPostgreSQLShowIndexAction("", params)
+		ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+		defer cancel()
+
+		b, err := a.Run(ctx)
+		require.NoError(t, err)
+		t.Logf("Full JSON:\n%s", b)
```
Comment (percona-csalguero) [inline actions/postgresql_show_index_action_test.go:73]: maybe we can delete this.

**Review by percona-csalguero — APPROVED** [review]

File: actions/postgresql_show_index_action_test.go (hunk 2 above)
Comment (BupycHuk) [reply actions/postgresql_show_index_action_test.go:73]: Yeah, but let's do it after release.

**Pull request merged**